apps/site-nft/src/translations/en.json

```diff
@@ -0,0 +1,43 @@
+{
+  "'/BZiQv'": "Upload Failed",
+  "'/lGbT2'": "Load an account stored online",
```

Comment:
This is going to be complicated for translation with no context.

Reply:
It needs it's descriptions.  It also needs to be in site-base, so we don't duplicate the translations.

https://github.com/thecointech/thecoin/issues/215

Reply:
https://github.com/thecointech/thecoin/issues/216